run.ps1

```diff
@@ -0,0 +1,30 @@
+# set the base tools directory
+$toolsLocalPath = Join-Path (Get-Location) ".tools"
```

Comment:
`Get-Location` will give the current working directory, but I think you want $PSScriptRoot so that it gets the folder that "run.ps1" is in, even if I call run from a different folder?